src/client/common/cy/revisions/generateImages.js

```diff
@@ -0,0 +1,15 @@
+const apiCaller = require('../../../services/apiCaller/');
+
+/* generateImages(layout, graph)
+   Generate a collection of cytoscape images that represent the different applied layouts
+   - Requires a valid layout list from the server and a valid cytoscape graph object
+   - Layouts must be formatted as a object containing postions, dates, and ids
+   - Graph must be a valid cytoscape json produced by cy.json();
+   - No images will be returned if there are no layouts. 
+*/
+function generateImages(layouts, graph) {
+  graph = graph.nodes.concat(graph.edges);
+  return apiCaller.renderImages({layouts, graph});
```

Comment:
This should send the number of images to render.  Instead of hardcoding it in the server api, it should take in n as a parameter.

Reply:
`generateImages()`, already bases number of images to render based on the length of the `layouts` array. The `layouts` array is produced by the `getRecentLayouts()` function, which now has a modifiable number of entries field.